src/Entity/DomNotification.php

```diff
@@ -349,5 +349,22 @@ private function resolveEmptyUser(UserInterface $user = NULL) {
     $account = \Drupal::entityTypeManager()->getStorage('user')->load(\Drupal::currentUser()->id());
     return $account;
   }
+  
+  /**
+   * Get a list of all recipients.
+   *
+   * @return array
+   */
+  public function getRecipients() {
+    $recipients = [];
+
+    if ($this->hasField('recipients')) {
+      foreach ($this->get('recipients')->getValue() as $user) {
+        $recipients[$user['target_id']] = $user['target_id'];
+      }
+    }
+
+    return $recipients;
+  }
```

Comment:
Please remove that field at all. Add this method to interface and implement it here using DB query to `dom_notifications_user_channels` table. Something like that:

```
return  \Drupal::database()
  ->select('dom_notifications_user_channels', 'dnuc')
  ->fields('dnuc', ['uid'])
  ->condition('channel_id', $this->getChannelID())
  ->execute()
  ->fetchCol();
```